### PR TITLE
Added GitHub release version number in footer

### PR DESCRIPTION
**Pull request by harrybasra95**
### Feature details
Added the version tag name from github releases it in the footer

### Feature image
image

### General explanation
Created a `fetchVersionNumber ` function to get the version number from the github releases api and added it to a state in `footer.tsx` file.

### Files changed
1. [footer.tsx](https://github.com/harrybasra95/hirosystems-explorer/blob/34562a7a592defebd9f9e8ec5665c5ae98281328/src/components/footer.tsx) 

     a. Added a useState to hold the version number info.
     b. Added useEffect to call the fetchVersionNumber function.

### Files added
1. [fetchVersionNumber.ts](https://github.com/harrybasra95/hirosystems-explorer/blob/6afaa5c18182452e2170488e0aa6578997524521/src/app/common/server-calls/fetchVersionNumber.ts)

### CODE REVIEW

File: src/app/common/server-calls/fetchVersionNumber.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export async function fetchVersionNumber(): Promise<string> {
+  const response = await fetch('https://api.github.com/repos/hirosystems/explorer/releases');
```
Comment (diwakergupta) [inline src/app/common/server-calls/fetchVersionNumber.ts:2]: This seems like the wrong behavior? This code will always report the latest release, even if an older version of the code is deployed. The version should be statically populated when a release is tagged IMO.

**Comment by He1DAr** [comment]
Thank you for your contribution @harrybasra95! Instead of using the last GitHub tag, you can try passing the release version as build arg in workflow yml. This way it will always reflect the currently deployed version.

**Comment by harrybasra95** [comment]
Hey @He1DAr, I have changed the approach, now I am using the workflow yml to get the release tag. Please review it once.
cc: @diwakergupta

**Review by He1DAr — APPROVED** [review]
LGTM. Thank you @harrybasra95!

**Comment by He1DAr** [comment]
@harrybasra95 could you please look into the failed checks so we merge this one?

**Comment by harrybasra95** [comment]
@He1DAr Fixed the linting issue.

**Comment by harrybasra95** [comment]
@He1DAr  Can you suggest some other issues I can work on?

**Comment by He1DAr** [comment]
@harrybasra95 It seems like github.event.release.tag_name is not populating as expected. When you have some time, would you mind taking a look and perhaps exploring some other alternatives? Thanks a bunch!

**Comment by harrybasra95** [comment]
@He1DAr  Can I get some logs so that I can get an idea of what is going wrong?

**Comment by andresgalante** [comment]
@harrybasra95 thanks for the contribution!! 🎉 

> Can you suggest some other issues I can work on?

There are a some of[ issues tagged as `good first issue` ](https://github.com/hirosystems/explorer/labels/good%20first%20issue)that you can take on, else if any of the issues pick your interest and you need more info, please let me know and I am happy to help with more context.

**Comment by babonyx** [comment]
I don't see the version information on https://explorer.hiro.so/

Was this reverted? Please reopen the issue.

**Comment by andresgalante** [comment]
@babonyx yes, due to https://github.com/hirosystems/explorer/pull/1119#issuecomment-1546014253

**Pull request merged**